io-engine/src/bin/io-engine-client/bdev_cli.rs

```diff
@@ -45,7 +45,7 @@ pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
         .arg(
             Arg::with_name("protocol")
                 .short("p")
-                .help("the protocol to used to share the given bdev")
+                .help("the protocol to used to share the given bdev. 0 for Off, 1 for Nvmf")
```

Comment:
What's the reason for replacing protocol name with 0 or 1 ? IMHO, it's nicer to have 'nvmf' instead of '1'.

Reply:
I see it expects i32 as of now for protocol. It was String previously. Need to check why this change was made.

Reply:
We expect Nvmf, off from client argument now.